dev/examples/manifold/klein.clj

```diff
@@ -67,18 +62,31 @@
 
 ;; Can you find the Emmy logo hidden inside?
 
-(ev/with-let [!r {:r 2}]
+(ev/with-let [!r {:r 2 :theta_0 1 :alpha_0 0 :steps 20}]
   [:<>
-   (leva/controls {:atom !r
-                   :folder {:name "Klein Bagel"}
-                   :schema {:r {:min 2 :max 6 :step 0.01}}})
+   (leva/controls
+    {:atom !r
+     :folder {:name "Klein Bagel"}
+     :schema {:r {:min 2 :max 6 :step 0.01}
+              :theta_0 {:label (->infix 'theta_0) :min 0 :max Math/PI :step 0.02}
+              :alpha_0 {:label (->infix 'alpha_0) :min 0 :max Math/PI :step 0.02}
+              :steps   {:min 500 :max 9000 :step 50}}})
    (slider-surface
     "Klein Bagel"
     {:f (ev/with-params {:atom !r :params [:r]}
           klein-bagel)
-     :opacity 0.75
      :u [0 (* 2 Math/PI)]
-     :v [0 (* 2 Math/PI)]})])
+     :v [0 (* 2 Math/PI)]}
+
+    (ph/geodesic
+     {:x0 [(ev/get !r :theta_0) 0]
+      :v0 [(list 'Math/cos (ev/get !r :alpha_0))
```

Comment:
@littleredcomputer agree... I think something like `ev/with-params` that compiled its body would be great. We can't use our generics trick here because of the `get` and `get-in` operations. Or maybe we can? We have enough test cases now that we should do something good here.